prow/cluster/jobs/istio/tools/istio.tools.yaml

```diff
@@ -0,0 +1,74 @@
+presubmits:
+
+  istio/tools:
+
+    - name: tools-lint
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - make
+              - lint
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+    - name: tools-build
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - go
+              - build
+              - ./...
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: build-pool
+
+    - name: tools-test
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - go
+              - test
+              - -race
+              - ./...
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+    - name: tools-fmtpy
+      path_alias: istio.io/tools
+      decorate: true
+      always_run: true
+      optional: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-06-13
+            command:
+              - make
+              - fmtpy-checkonly
```

Comment:
this may not work as `make fmtpy-checkonly` may require user input:

```
sdake@beast-01:~/go/src/istio.io/tools$ make fmtpy-checkonly
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following package was automatically installed and is no longer required:
  libllvm5.0
Use 'sudo apt autoremove' to remove it.
The following additional packages will be installed:
  libpython3-dev libpython3.5-dev python3-dev python3-setuptools python3-wheel python3.5-dev
Suggested packages:
  python-setuptools-doc
The following NEW packages will be installed:
  libpython3-dev libpython3.5-dev python3-dev python3-pip python3-setuptools python3-wheel python3.5-dev
0 upgraded, 7 newly installed, 0 to remove and 34 not upgraded.
Need to get 38.0 MB of archives.
After this operation, 55.2 MB of additional disk space will be used.
Do you want to continue? [Y/n]
After this operation, 55.2 MB of additional disk space will be used.
Do you want to continue? [Y/n]
```

are the deps in the image?